docs/docsite/rst/command_guide/index.rst

```diff
@@ -18,4 +18,11 @@ Ansible provides ad hoc commands and several utilities for performing various op
 
    intro_adhoc
    command_line_tools
-   cheatsheet
\ No newline at end of file
+   cheatsheet
+
+.. seealso::
+
+   `Ansible Navigator <https://ansible.readthedocs.io/projects/navigator/>`_
+       A command-line tool and a TUI that provides a convenient user interface for most
+       of the native Ansible command-line utilities and allows to run Ansible automation content
+       inside containers (`execution environments <https://docs.ansible.com/ansible/devel/getting_started_ee/index.html>`_)
```

Comment:
This can probably be linked via native references. Something like this will probably work:
```suggestion
       inside containers (:doc:`execution environments <getting_started_ee/index>`)
```

Reply:
It's actually :ref: we want to use here with the label from the top of the file... so ends up
:ref:\`execution environments <getting_started_ee_index>\`

Reply:
@samccann `:ref:` requires an explicit label to be maintained. `:doc:` can refer to the document w/o labels.

Reply:
By the way, with this change, the linked document should be unorphaned: https://github.com/ansible/ansible-documentation/blob/6369e7a/docs/docsite/rst/getting_started_ee/index.rst?plain=1#L1-L2.

Reply:
We chose :ref: a long time ago in the ansible docs and removed :docs:. So let's not open that back up please.

And yes good point - we can remove orphan now.